src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
-export interface Figure {
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
 
+enum Color {
+  red,
```

Comment:
В enum пиши поля з великої літери і чому в тебе не задано значення для цих кольорів, а просто 0, 1, 2,
```suggestion
  Red,
```